apps/civil/civil-service/perftest.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   timeout: 10m
   values:
     java:
-      image: hmctspublic.azurecr.io/civil/service:pr-7197-517e756-20251127080432 #{"$imagepolicy": "flux-system:perftest-civil-service"}
```

Comment:
if you want to deploy the prod image it is probably best to only change the image name to civil-service, you don't need to change the image itself to prod-c2d00a4-...